packages/ui/src/components/item-layout/item-layout.native.stories.tsx

```diff
@@ -23,11 +22,8 @@ export default meta;
 
 export const ItemLayoutStory = {
   args: {
+    actionIcon: <ChevronRightIcon color="ink.action-primary-default" variant="small" />,
     captionLeft: 'captionLeft',
-    captionRight: 'captionRight',
```

Comment:
Any reason to remove these? I added them to give the full picture of what `ItemLayout` can show

Reply:
Not sure why I removed, maybe was during some testing so I can add back. Tbh, this is what I encourntered with `ItemLayout` in storybook, so for me it just wasn't working as expected when I used in `ThemeSwitcher`...

![image](https://github.com/user-attachments/assets/5b18718c-a08f-44bb-ad18-48f450751419)